src/styles/cards-block/card.scss

```diff
@@ -0,0 +1,137 @@
+.conatiner {
+  width: 1024px;
+  margin: 0 auto;
+  padding: 50px 40px;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(var(--column-count), 248px);
+}
+
+@media (min-width: 488px) {
+  .conatiner{
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .conatiner{
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .conatiner{
+    --column-count: 4;
+  }
+}
+
+@media (min-width: 1200px) {
+  .conatiner{
+    width: 1200px;
+  }
+}
+
+.card  {
+  margin: 0 auto;
+  padding: 32px 16px 16px;
```

Comment:
Margin is defined twice for the '.card' class. Consider removing one of them to avoid confusion.